csrankings-s.csv

```diff
@@ -812,6 +812,7 @@ Shaifu Gupta,IIT Jammu,https://www.iitjammu.ac.in/computer_science_engineering/f
 Shaileshh Bojja Venkatakrishnan,Ohio State University,https://sites.google.com/site/shaileshhbv,NOSCHOLARPAGE
 Shailey Minocha,Open University UK,http://stem.open.ac.uk/people/sm577,NOSCHOLARPAGE
 Shalabh Bhatnagar,IISc Bangalore,http://www.csa.iisc.ac.in/~shalabh,cj3fJJsbjAoC
+Shalabh Gupta,University of Connecticut,https://linkslab.engr.uconn.edu/guptabio/,31ivHAQAAAAJ

```

Comment:
EE, not CS.

Reply:
Dr. Shalabh Gupta is from ECE of UConn. He is the director of [Intelligent Networked Systems and Robotics](https://linkslab.engr.uconn.edu/). He also have several publications in ICRA. Can you please include him too?

Reply:
Criteria are clear (has nothing to do with pubs at all, you checked the box indicating you read this):

> Eligible faculty include only full-time, tenure-track research
> faculty members on a given campus who can solely advise PhD students in
> Computer Science. Faculty not in a CS department or similar who can
> advise PhD students in CS can be included regardless of their home
> department. Provide justification, pointing to specific home pages
> showing how faculty not in a CS department meet the inclusion criteria,
> e.g. showing a courtesy appointment in CS.

Reply:
I see. Thanks for pointing this out. How can I remove them from this pull request? Can you please advise?